internal/graphapi/pool_test.go

```diff
@@ -212,6 +241,22 @@ func TestMutate_PoolUpdate(t *testing.T) {
 			},
 			errorMsg: "validator failed",
 		},
+		{
+			TestName: "fails to update pool with port with conflicting OwnerID",
+			Input: graphclient.UpdateLoadBalancerPoolInput{
+				Name:       newString("ImaPool"),
+				AddPortIDs: []gidx.PrefixedID{port.ID},
```

Comment:
Im assuming you are talking about UpdateLoadBalancerInput since we dont really mutate ports through the load balancer since ports dont exist without loadbalancers, and instead to mutate ports directly instead?

Reply:
Also could the same be said with pools and origins since origins dont exist outside of pools and thus we shouldnt mutate origins through pools as well?

Reply:
Also could the same be said with pools and origins since origins dont exist outside of pools and thus we shouldnt mutate origins through pools as well?